### PR TITLE
Test cleanups

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: tests/commands/dump.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 
 @pytest.mark.cmd
+@pytest.mark.django_db
 def test_dump_noargs():
     """Dump requires an output option."""
     with pytest.raises(CommandError) as e:
```
Comment (dwaynebailey) [inline tests/commands/dump.py]: This is just a test of the command options so no DB needed.

File: tests/commands/merge_user.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 
 @pytest.mark.cmd
+@pytest.mark.django_db
 def test_purge_user_nousers():
     with pytest.raises(CommandError) as e:
         call_command('merge_user')
```
Comment (dwaynebailey) [inline tests/commands/merge_user.py]: Ditto no DB needed.

File: tests/commands/merge_user.py [hunk 3]
```diff
@@ -20,6 +21,7 @@ def test_purge_user_nousers():
 
 
 @pytest.mark.cmd
+@pytest.mark.django_db
 def test_purge_user_only_one_user(member):
     with pytest.raises(CommandError) as e:
```
Comment (dwaynebailey) [inline tests/commands/merge_user.py]: No DB needed.

File: tests/commands/dump.py (hunk 1 above)
Comment (phlax) [inline tests/commands/dump.py]: hmmm - on im not sure - im backporting the things that were failing once we remove the db marker from the fixtures - and im pretty sure this was one of them.

i can check

File: tests/commands/merge_user.py (hunk 2 above)
Comment (phlax) [inline tests/commands/merge_user.py]: ditto

File: tests/commands/merge_user.py (hunk 3 above)
Comment (phlax) [inline tests/commands/merge_user.py]: ditto

Comment (dwaynebailey) [inline tests/commands/merge_user.py]: My bad I see that it has 'member'

Comment (phlax) [inline tests/commands/merge_user.py]: afaict ~all tests need db

**Pull request merged**